flask_app/templates/base.html

```diff
@@ -4,8 +4,8 @@
 <div class="container">
 <nav>
   <ul>
-      <li><a href="/download">Show Code</a>
-      <li><a href="/data">Upload Data</a>
+      <!--<li><a href="/download">Show Code</a>-->
+      <!--<li><a href="/data">Upload Data</a>-->
```

Comment:
@tyler-mcmahon @KD335: please remove commented out code before we merge it to main.